package-lock.json

```diff
@@ -1,8 +1,5533 @@
 {
```

Comment:
Lock should not be updated in the PR, only syntax and generating script should be

Reply:
I'm not even sure how that happen D: , my JS-packaging knowledge is limited and I use yarn too. I thought package-lock.json never changed suddenly too.
I just replaced package.json and package-lock.json with the ones from the repo.